Library/Models/SurveyDownloadDataRequest.cs

```diff
@@ -94,5 +99,11 @@ public class SurveyDownloadDataRequest
         /// May be null or empty if no end date is specified.
         /// </summary>
         public string EndDate { get; set; }
+
+        /// <summary>
+        /// The survey version (Etag)
+        /// May be null or 0 if no survey version is specified.
+        /// </summary>
+        public long? SurveyVersion { get; set; }
```

Comment:
Why is this a long instead of a string?

Reply:
It is the the etag, which is a number

Reply:
In the database the Etag is a long int

Reply:
I think the endpoint Carlos is creating (to retreive the list of etag, is returning an string (converting the long to string). Not sure which option is better, but both should have the same type (both long or both string)